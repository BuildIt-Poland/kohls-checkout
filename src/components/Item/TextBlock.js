import styled from 'styled-components';

import { spacingSmall } from '../../styles/designTokens';

const TextBlock = styled.div`
  margin-bottom: ${spacingSmall};

  &:last-child {
    margin-bottom: 0;
  }
`;

export default TextBlock;
