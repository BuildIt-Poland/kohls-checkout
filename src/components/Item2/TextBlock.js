import styled from 'styled-components';

import { spacingSmall } from '../../styles/designTokens';

const TextBlock = styled.div`
  margin-bottom: ${spacingSmall};
`;

export default TextBlock;
