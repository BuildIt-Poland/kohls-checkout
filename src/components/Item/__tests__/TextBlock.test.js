import React from 'react';
import { create } from 'react-test-renderer';

import TextBlock from '../TextBlock';

describe('COMPONENT - Item TextBlock', () => {
  it('renders correctly', () => {
    const component = create(<TextBlock />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
