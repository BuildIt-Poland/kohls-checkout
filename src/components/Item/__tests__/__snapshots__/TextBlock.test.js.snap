// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Item TextBlock renders correctly 1`] = `
.c0 {
  margin-bottom: 1rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<div
  className="c0"
/>
`;
